src/streaming/controllers/AlternativeMpdController.js

```diff
@@ -258,7 +270,8 @@ function AlternativeMpdController() {
                 returnOffset: parseInt(alternativeMpdNode.returnOffset || '0', 10) / 1000,
                 triggered: false,
                 watched: false,
```

Comment:
What is the diff between those attributes?
Instead of watched I will use `completed` because is more generic for an event, not every event is refering to watch a video.